packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -41,19 +41,39 @@
   @include mdc-text-field-helper-text-color($mdc-text-field-helper-text-color);
   @include mdc-text-field-fullwidth-bottom-line-color($mdc-text-field-fullwidth-bottom-line-color);
   @include mdc-text-field-icon-color($mdc-text-field-icon-color);
+  @include mdc-ripple-surface;
```

Comment:
nit: move all non-`mdc-text-field-*` mixins (i.e. `mdc-ripple-*`, `mdc-states-*`, and `mdc-floating-label-*` mixins) to be first, so it's more clear which mixins are from another package and which mixins are from this one.

Reply:
Done!